evaluator/src/app/shared/components/footer-credits/footer-credits.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="body">
     <p><i class="fa-solid fa-code"></i><span class="bold"> Desenvolvido por: </span>
     <a href="https://www.linkedin.com/in/caiodsj">Caio Diego</a> | 
-    <a href="">Helena Luz</a> | 
-    <a href="">Raphael Theodoro</a> | 
-    <a href="">Robbinson</a> | 
-    <a href="">Rafael R</a> | 
-    <a href="">Maria Cláudia</a>
+    <a href="https://www.linkedin.com/in/helena-luz-139430214/">Helena Luz</a> | 
+    <a href="https://www.linkedin.com/in/raphael-cerutti-teodoro">Raphael C. Teodoro</a> | 
+    <a href="https://www.linkedin.com/in/rbcaputo">Rob Caputo</a> | 
+    <a href="https://www.linkedin.com/in/rafael-demarch/">Rafael R. Demarch</a> | 
+    <a href="">Maria Claudia</a>
```

Comment:
Add linkedin da Claudia, só tem que ver qual nome ela prefere apresentar ali depois:
```suggestion
    <a href="https://www.linkedin.com/in/itisclaudia/">Maria Claudia</a>
```